win_build.sh

```diff
@@ -28,7 +28,9 @@ else
       sys_resource \
       sys_wait \
       setlocale \
-      strtok_r
+      strtok_r \
+      poll \
+      recv
```

Comment:
If we're going to use select() instead of recv() as Corey suggested, this dependency should also be removed.